integration-tests/dataformats-json/src/main/java/org/apache/camel/quarkus/component/dataformats/jackson/json/JacksonJsonResource.java

```diff
@@ -0,0 +1,692 @@
+        String marshalledAsString = context.getTypeConverter().convertTo(String.class, marshalled);
+        // we enable the wrap root type feature so we should have TestPojoView
+        assertEquals("{\"TestPojoView\":{\"age\":30,\"height\":190,\"weight\":70}}", marshalledAsString);
+    }
+
+    @Path("jackson/concurrent")
+    @GET
+    public void jacksonConcurrent() throws Exception {
+        doSendMessages(10, 5);
+    }
+
+    @Path("jackson/unmarshal-listjackson")
+    @GET
+    public void jacksonUnmarshalListJackson(String body) throws Exception {
+
+        MockEndpoint mock = context.getEndpoint("mock:jackson-list-unmarshal-reversePojo", MockEndpoint.class);
+        mock.expectedMessageCount(1);
+        mock.message(0).body().isInstanceOf(List.class);
+
+        String json = "[{\"name\":\"Camel\"}, {\"name\":\"World\"}]";
+        producerTemplate.sendBody("direct:jackson-list-unmarshal-backPojo", json);
+
+        mock.assertIsSatisfied();
+
+        List list = mock.getReceivedExchanges().get(0).getIn().getBody(List.class);
+        assertNotNull(list);
+        assertEquals(2, list.size());
+
+        TestPojo pojo = (TestPojo) list.get(0);
+        assertEquals("Camel", pojo.getName());
+        pojo = (TestPojo) list.get(1);
+        assertEquals("World", pojo.getName());
+
+        mock.expectedMessageCount(2);
+        mock.message(1).body().isInstanceOf(List.class);
+
+        json = "[{\"name\":\"Camel\"}]";
+        producerTemplate.sendBody("direct:jackson-list-unmarshal-backPojo", json);
+
+        mock.assertIsSatisfied();
+
+        list = mock.getReceivedExchanges().get(1).getIn().getBody(List.class);
+        assertNotNull(list);
+        assertEquals(1, list.size());
+
+        pojo = (TestPojo) list.get(0);
+        assertEquals("Camel", pojo.getName());
+
+    }
+
+    @Path("jackson/conversion-pojo")
+    @GET
+    public void jacksonConversionPojo(String body) throws Exception {
+        synchronized (context) {
+            context.getGlobalOptions().put(JacksonConstants.ENABLE_TYPE_CONVERTER, "true");
+            context.getGlobalOptions().put(JacksonConstants.TYPE_CONVERTER_TO_POJO, "true");
+
+            Order order = new Order();
+            order.setAmount(1);
+            order.setCustomerName("Acme");
+            order.setPartName("Camel");
+
+            String json = (String) producerTemplate.requestBody("direct:jackson-conversion-pojo-test", order);
+            assertEquals("{\"id\":0,\"partName\":\"Camel\",\"amount\":1,\"customerName\":\"Acme\"}", json);
+
+            context.getGlobalOptions().put(JacksonConstants.TYPE_CONVERTER_MODULE_CLASS_NAMES,
+                    JaxbAnnotationModule.class.getName());
+
+            order = new Order();
+            order.setAmount(1);
+            order.setCustomerName("Acme");
+            order.setPartName("Camel");
+
+            json = (String) producerTemplate.requestBody("direct:jackson-conversion-pojo-test", order);
+            /*
```

Comment:
So we need a follow up issue in camel-quarkus or maybe camel ?

Reply:
Yes, that's my plan

Reply:
yeah, it's good to have the link to the follow up issue in the code or no comment at all.